rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1925,9 +1925,10 @@ void game_list_frame::BatchCreateCPUCaches()
 		return;
 	}
 
-	const QString main_label = tr("Creating all PPU caches");
+	const QString main_label = tr("Creating all LLVM caches");
 
-	progress_dialog* pdlg = new progress_dialog(tr("PPU Cache Batch Creation"), main_label, tr("Cancel"), 0, total, true, this);
+	progress_dialog* pdlg = new progress_dialog(tr("LLVM Cache Batch Creation"), main_label, tr("Cancel"), 0, total, true, this);
+	pdlg->setWindowFlags(Qt::Window | Qt::WindowMinimizeButtonHint);
```

Comment:
When testing this I had multiple applications open.
When clicking minimize on the Batch compiler dialog it'd minimize the batch compiler but wouldn't minimize the main window.
What was happening was the focus would switch to other applications. But the desktop could not be reached.
It seems we can change the event listener to minimize the main application when the batch compiler minimizes and maximize the main window when the focus is restored to the batch compiler.

Reply:
I don't understand why you are trying to reinvent the wheel.
It seems like you are trying to fix an "issue" that anyone can simply circumvent by using alt+tab or by clicking on the desktop button on the bottom right

Reply:
Yea, I think it's better to just remove the change.